keps/sig-node/4622-topologymanager-max-allowable-numa-nodes/README.md

```diff
@@ -312,16 +335,24 @@ you need any help or guidance.
 <!--
 This section must be completed when targeting alpha to a release.
 -->
+
 1.31:
+
 - enable by default
 - allow gate to disable the feature
 - release note
 
-1.32:
+1.34:
+
 - promote to GA
-- cannot be disabled
+- LockToDefault: true (cannot be disabled)
 - release note
 
+1.35:
+
+- feature gate removed
```

Comment:
is the feature gate being removed or are we allowing this option independent of whether the featuregate is on?